src/cls/IPM/Utils/Update/VersionBase.cls

```diff
@@ -0,0 +1,52 @@
+Class %IPM.Utils.Update.VersionBase [ Abstract ]
+{
+
+/// Returns dynamic array of update method names to apply in order that can be run on both primary and backup
+/// Example output list: ["methodA", "methodB", "methodC"]
+/// When mirror enabled: These steps will run on all members
+/// When mirror not enabled: These steps will be run on the instance
+ClassMethod GetOrderedMethodsAnyMirrorMember() As %Library.DynamicArray [ Abstract ]
+{
+}
+
+/// Returns dynamic array of update method names to apply in order that must be run on the primary
```

Comment:
Similarly, update the description to:
```
Returns a dynamic array of update method names to apply in order. This method has different behavior when the instance is mirrored versus not. <br />
When the instance is not mirrored, these are always run synchronously when the zpm "update" command is run (during the ApplyUpdateSteps lifecycle phase). <br />
When the instance is mirrored, this will run only when the instance becomes the primary member. <br />
<example>
   Output: ["methodA", "methodB", "methodC"]
</example>
```